src/item.cpp

```diff
@@ -2493,16 +2493,11 @@ int item::bash_resist() const
         return resist;
     }
     if (item::item_tags.count("leather_padded") > 0){
-        l_padding = volume() * (float(get_coverage()) / 100) / 2.5;
-        if (l_padding > 5 ){
-            l_padding = l_padding / 2.5;   //Hard cap so coats don't become solid steel
-        }
+               //Max value           Steepness of graph         Center of 'S'
```

Comment:
Instead of adding a name in the comment why put the values into constants with a useful name:
```C++
static constexpr float max_value = 4.0f;
static constexpr float stepness = -0.8f;
static constexpr float center_of_S = 2.5f;
l_padding = max_value / (1 + exp(stepness * ( get_thickness() - center_of_S)));
```
Not only is the formula easier to read, but it also prevents the comment from getting out of sync with the actual code and changing the values later does not interfere with the actual formula.